### ISSUE TITLE
More options menu: Accessible name of the version history modal

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#2023]
## Bug Report

**Problematic behavior**
The accessible name of the modal is in English when it should be translated to French if the user's chosen interface is in French.

**Expected behavior/code**
```

<div aria-label="version history modal" class="sc-5b83f41b-0 hYDwen --docs--modal-select-version noPadding">
    <h1 id="modal-select-version-title" class="sc-5b83f41b-0 sc-458c56ea-0 hLzqHY dUDhBq sr-only">Historique des versions</h1>

```

**Possible Solution**
Supprimer l’attribut `aria-label="version history modal"`, ajouter un `attribut aria-labelledby="modal-select-version-title"` dans l’élément `div role="dialog"`

**Additional context/Screenshots**

<img width="882" height="181" alt="Image" src="https://github.com/user-attachments/assets/a59653b2-0ef1-4f5d-a787-e43226318cd2" />

**Critères RGAA**

Critère 7.1 : [Chaque script est-il, si nécessaire, compatible avec les technologies d'assistance ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#7.1)
**Impact** 

A French user with a screen reader who hears "version history modal" in English when the modal opens instead of "Historique des versions" is confusing.
